tests/rados/pool_tests.py

```diff
@@ -1078,11 +1118,7 @@ def run(ceph_cluster, **kw):
                 log.info(
                     f"Post modification, Pg set fetched from pool id {pool_id} for pg id {pgid} is {new_pg_set}"
                 )
-                log.info(f"Current OSDs in acting set {new_pg_set}")
-                log.info(
-                    f"Initial pg set fetched from pool id {pool_id} for pg id {pgid} is {pg_set}"
-                )
-                if len(new_pg_set) == 1 and new_pg_set[0] == pg_set[0]:
+                if len(new_pg_set) == 1:
```

Comment:
Is there a reason to remove the `new_pg_set[0] == pg_set[0]` check?

Reply:
Yes. We are no longer using pop() to get the OSDs for bringing down. So new_pg_set[0] might not be same as pg_set[0].

Also, this check is not required in the test here. let me know if you think otherwise.